### PR TITLE
Ewlj 80 basic page prototype

### PR DESCRIPTION
**Pull request by aCyborg**
## Ticket(s)

**Github Issue**

N/A

**Jira Ticket**

- [EWLj-80: Basic Page Prototype](https://issues.ama-assn.org/browse/EWLJ-80)

## Description

Create a podcast detail page. This includes:
* Page hero atom
* Page header molecule
* Rail nav Molecule
* Two-column-left--simple layout

## To Test

- [ ] Spin up Patternlab
- [ ] Review the Page hero atom
- [ ] Review the header molecule
- [ ] Review the Rail nav Molecule
- [ ] Review Two-column-left--simple layout
- [ ] Review the Basic Page Page

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

### CODE REVIEW

**Comment by gleroux02** [comment]
@aCyborg seeing problems with our favorite component the share links between 540px and 600px:
image

File: styleguide/source/_patterns/02-molecules/rail-nav.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+This Pattern shows the rail nav, which appears on basic pages.
+
+### Variables
+~~~
+rail_navs {
+  url:
+    type: string / required
+  text:
+    type: string / required
+  childs:
```
Comment (gleroux02) [inline styleguide/source/_patterns/02-molecules/rail-nav.md:11]: children?

**Review by gleroux02 — CHANGES_REQUESTED** [review]
Just the share issue and childs :)

**Comment by aCyborg** [comment]
@gleroux02 I've updated the childs to be children, but I'm going to leave the social icons as they are for now since I completely re-thought social sharing in my next PR. Once this gets merged in, we can take a closer look at this social setup in that PR.

**Review by gleroux02 — APPROVED** [review]
👍  sounds good about the share links.

**Pull request merged**